bin/run-test.sh

```diff
@@ -1,18 +1,56 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+# Usage: -c to report coverage
+
+while true; do
+  case $1 in
+    -c)
+      cover=1
+      ;;
+
+    *)
+      break
+  esac
+
+  shift
+done
+
+# Lint
 jshint . || exit 1
+
+# Install framework deps
 for dir in test/hooks/fixtures/*/ ;
 do
   echo -en "travis_fold:start:npm_install_${dir}\\r" | tr / _
   echo "npm install in ${dir}"
-  cd ${dir} && npm install && cd -
+  cd "${dir}" && npm install && cd - || exit
```

Comment:
It would be better to do it this way:
```bash
(cd ${dir}; npm install) || exit 1
```
Parentheses give you a sub-shell, so you don't have to cd back.